### ISSUE TITLE
Sitewide: make site responsive/available on mobile 

### ISSUE THREAD

**Issue by kimpham54** [dulibrarytech/digitaldu-frontend#60]
In addition to the testing sessions, D&D staff tested the tool on mobile devices. The landing page has numerous issues – the Browse By/Facets, Collections, and site name have issues on all pages where they are found (landing page and results pages). The search box on a results page cuts off on the left, so the user cannot see what they are typing in. The way that results show makes entries really long when there is a long description, and the numbers for search page results is far on the left (for examples, use a sample search “Jewish”).The information on the View Details pages is cut off. The footer is off center.
